lib/dor/was_seed/thumbnail_generator_service.rb

```diff
@@ -5,21 +5,23 @@
 
 module Dor
   module WasSeed
+    # TODO: why is this a class if it is full of class methods and has no constructor?
     class ThumbnailGeneratorService
       # because this date is earlier than any of the archived dates of the content,
       # this tells the wayback machine to provide the earliest capture date.
```

Comment:
Is this comment bogus now that we dumped the date const?

Reply:
Yes, thanks!